### ISSUE TITLE
Add keyboard navigation to JingetDropDown and JingetDropDownTreeList

### ISSUE THREAD

**Issue by VahidFarahmandian** [VahidFarahmandian/Jinget#96]
**Is your feature request related to a problem? Please describe.**
The user should be able to easily move between the items using the keyboard and select the desired option.

**Describe the solution you'd like**
It is preferable to use the facilities & utilities in the select2.js

**Comment by VahidFarahmandian** [comment]
> **Is your feature request related to a problem? Please describe.** The user should be able to easily move between the items using the keyboard and select the desired option.
> 
> **Describe the solution you'd like** It is preferable to use the facilities & utilities in the select2.js

`JingetDropDownList` already supports the keyboard navigation & this feature should be applied for `JingetDropDownTreeList` only, as this component render the items manually.
